packages/fcl/src/exec/query.js

```diff
@@ -1,31 +1,31 @@
-import {invariant} from "@onflow/util-invariant"
 import * as sdk from "@onflow/sdk"
-import {isRequired, isObject, isString} from "./utils/is"
 import {normalizeArgs} from "./utils/normalize-args"
 import {prepTemplateOpts} from "./utils/prep-template-opts.js"
 import {preQuery} from "./utils/pre.js"
 
-/** Query the Flow Blockchain
+/**
+ * @description
+ * Allows you to submit scripts to query the blockchain.
  *
- *  @arg {Object} opts         - Query Options and configuration
- *  @arg {string} opts.cadence - Cadence Script used to query Flow
- *  @arg {ArgsFn} opts.args    - Arguments passed to cadence script
- *  @arg {Object} opts.template - Interaction Template for a script
- *  @arg {number} opts.limit   - Compute Limit for Query
+ *  @param {Object} opts         - Query Options and configuration
+ *  @param {string} opts.cadence - Cadence Script used to query Flow
```

Comment:
Why no brackets on this one?

Reply:
@justinbarry Cadence is required. Brackets make it optional.